src/pages/tasks.page.tsx

```diff
@@ -0,0 +1,175 @@
+import {BsFillPenFill} from 'react-icons/bs';
+import { AiOutlineArrowLeft } from 'react-icons/ai'
+import { IconContext } from 'react-icons'
+import Link from 'next/link'
+import styled from '@emotion/styled'
+import React, { useRef, useState, useEffect } from 'react';
+
+
+const tasks = [{
+    taskID: 0,
+    taskName: '課題１ですよーーーーーーーーーーーーー', 
+    taskDueYear: 2023, 
+    taskDueMonth: 6, 
+    taskDueDay: 18
+},{
+    taskID: 1,
+    taskName: "課題２",
+    taskDueYear: 2023,
+    taskDueMonth: 6,
+    taskDueDay: 23
+},{
+    taskID: 2,
+    taskName: "課題３",
+    taskDueYear: 2023,
+    taskDueMonth: 6,
+    taskDueDay: 30
+}
+]
+
+
+function LeftArrowButton(){
+    return(
+        <Link href="/">
+                <IconContext.Provider value={{size: '40px', color: '#fff'}}>
+                    <AiOutlineArrowLeft />
+                </IconContext.Provider>
+        </Link>
+    )
+}
+
+type TextProps = Omit<JSX.IntrinsicElements['span'], 'children'> & { children: string };
```

Comment:
ほかのところに合わせてここも `ComponentProps` にしたほうがよさそう :pray: (ちなみに Omit せずに & でつなげるだけでいけます
```suggestion
interface TextProps exdtends ComponentProps<'span'> {
  children: string
}
```